Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:20.04
+
```

Comment:
Better practise would be to inheirt from one of the python containers, then you wouldnt need to install python and you get much smaller images,